### DISCUSSION TITLE
No overlapping error

### DISCUSSION THREAD

**Discussion by BioinSAR** [zhengxwen/HIBAG#28] [Q&A]
Hello,

Thank you so much for your great package.

I am trying to impute the HLA data using HIBAG, I get error as "Error in hlaPredict(model, hapmap.ceu, type = "response") : 
  There is no overlapping of SNPs!?
  
Why I get this error and what should I do to fix this error? 

Thanks!

**Comment by zhengxwen** [comment]
Please use platform-specific models:
https://hibag.s3.amazonaws.com/hlares_index.html

**Reply by BioinSAR** [reply]
Thanks!
